parboiled-core/src/main/scala-2/org/parboiled2/support/OpTreeContext.scala

```diff
@@ -748,20 +749,42 @@ trait OpTreeContext[OpTreeCtx <: ParserMacros.ParserContext] {
       val pushBuilderResult: Tree,
       val pushSomePop: Tree,
       val pushNone: Tree
-  )
+  ) {
+    def withLazyBuilder: Collector = this
+  }
 
   lazy val rule0Collector = {
     val unit = q"()"
     new Collector(unit, unit, q"true", unit, unit)
   }
 
+  lazy val lazyRule1Collector = new Collector(
+    valBuilder = q"var builder: scala.collection.immutable.VectorBuilder[Any] = null",
```

Comment:
Maybe I should rename `valBuilder` to `declareBuilder`?

Reply:
If you want to.
But I think it's fine as it is as well.